common/table.js

```diff
@@ -1477,6 +1477,13 @@
                     UiUtils.attachFooterResizeSensor(0);
                 }
 
+                if (scope.vm.config.displayMode === recordsetDisplayModes.facetPopup) {
+                    var overflowContainerFacet = document.getElementById('overflow-container-facet');
```

Comment:
Selecting by id works but is fragile and you shouldn't do this here. We've designed recordset directive so it can be used multiple times on the page without assuming where. If you look at the `attachDOMElementsToScope`, we're creating a `scope.parentContainer` that will return the container of this instance of recordset. So you should use that and then select based on a class:

```js
scope.parentContainer.querySelector("<some selector>")
```

If you do that, then you don't need to do this twice once for popup and once for normal recordset. The newly added `id`s are not needed either.